src/screens/expandedAssetSheet/components/sections/marketSection/MarketStatsSection.tsx

```diff
@@ -1,24 +1,24 @@
 import React, { memo } from 'react';
 import * as i18n from '@/languages';
 import { Box } from '@/design-system';
-// import { MarketStatsCard } from './MarketStatsCard';
-import { AssetInfoList } from './AssetInfoList';
+import { MarketStatsCard } from './MarketStatsCard';
 import { CollapsibleSection, LAYOUT_ANIMATION } from '../../shared/CollapsibleSection';
 import { SheetSeparator } from '../../shared/Separator';
-import { SectionId } from '@/screens/expandedAssetSheet/context/ExpandedAssetSheetContext';
+import { SectionId, useExpandedAssetSheetContext } from '@/screens/expandedAssetSheet/context/ExpandedAssetSheetContext';
 import Animated from 'react-native-reanimated';
+import { useTokenMarketStats } from '@/resources/metadata/tokenStats';
 
 export const MarketStatsSection = memo(function MarketStatsSection() {
-  {
-    /* BLOCKED: blocked by backend for data */
-  }
-  {
-    /*content={<MarketStatsCard />} */
-  }
+  const { basicAsset: asset } = useExpandedAssetSheetContext();
+
+  const { data: marketData } = useTokenMarketStats({ chainID: asset.chainId, address: asset.address });
+
+  if (!marketData || Object.keys(marketData).length === 0) return null;
+
   return (
     <Box as={Animated.View} layout={LAYOUT_ANIMATION} gap={28}>
       <CollapsibleSection
-        content={<AssetInfoList />}
+        content={<MarketStatsCard marketData={marketData} />}
```

Comment:
Lost the token data list of market cap, valuation, etc. Should be this:

```tsx
        content={
          <Box gap={24}>
            <MarketStatsCard />
            <AssetInfoList />
          </Box>
        }
```

Because of that, should move the data fetching hook to the `MarketStatsCard` itself and return null there if doesn't exist.